test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -831,9 +839,8 @@
                       first))))))))
 
 (deftest datetime-diff-type-test
-  ;; FIXME — The excluded drivers below don't have TIME types. These shouldn't be hard-coded with #26807
-  (mt/test-drivers (disj (mt/normal-drivers-with-feature :datetime-diff)
-                         :oracle :presto :redshift :sparksql :snowflake)
+  (mt/test-drivers (->> (disj (mt/normal-drivers-with-feature :datetime-diff) :snowflake)
+                        (filter mt/supports-time-type?))
```

Comment:
Note snowflake coverage on this test will be added here https://github.com/metabase/metabase/pull/27068#top